src/main/java/sx/blah/discord/api/internal/Requests.java

```diff
@@ -122,8 +123,9 @@ private Request(Class<? extends HttpUriRequest> clazz, IDiscordClient client) {
 		 *
 		 * @throws RateLimitException
 		 * @throws DiscordException
+		 * @throws CloudFlareException
 		 */
-		public String makeRequest(String url, BasicNameValuePair... headers) throws RateLimitException, DiscordException {
+		public String makeRequest(String url, BasicNameValuePair... headers) throws RateLimitException, DiscordException, CloudFlareException {
```

Comment:
It is so obvious that you did not test this change in the *slightest*. This change is **extremely** breaking. You'd added a checked exception to one of the most commonly used methods in the entire library.... Open up *any* accessor of this method and you'll be shown a wall of errors for the uncaught exception.....